packages/css/src/components/form/form-group.scss

```diff
@@ -54,11 +55,15 @@
   legend.dso-label-container {
     border-bottom: 0;
     font-size: $font-size-base;
-    margin-bottom: $u1;
+    margin-bottom: 0;
```

Comment:
Deze CSS rule kan weg, wordt niet meer gebruikt en kan in de voorbeeldpagina worden omgeschreven